src/routers/filing.py

```diff
@@ -85,3 +85,18 @@ async def patch_filing(request: Request, lei: str, period_name: str, field_name:
 @requires("authenticated")
 async def update_task_state(request: Request, lei: str, period_name: str, task_name: str, state: StateUpdateDTO):
     await repo.update_task_state(request.state.db_session, lei, period_name, task_name, state.state, request.user)
+
+
+@router.get("/institutions/{lei}/filings/{period_name}/contact-info", response_model=ContactInfoDTO)
+@requires("authenticated")
+async def get_contact_info(request: Request):
+    result = await repo.get_contact_info(request.state.db_session)
+    if result:
+        return result
+    return JSONResponse(status_code=status.HTTP_204_NO_CONTENT, content=None)
```

Comment:
this doesn't look right, seems like it would just return the first record in the contact_info table, not one that relates to a filing.

Reply:
the parameters should follow the update contact info one, excluding the request payload; and we would filter based on lei and filing period